extensions/3DTILES_metadata/schema/3DTILES_metadata.namespaces.namespace.classes.class.property.schema.json

```diff
@@ -0,0 +1,106 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Class Property",
+  "type": "object",
+  "description": "A class property.",
+  "properties": {
+    "name": {
+      "type": "string",
+      "minLength": 1,
+      "description": "The name of the property, e.g. for display purposes. The string is UTF-8 encoded."
+    },
+    "description": {
+      "type": "string",
+      "minLength": 1,
+      "description": "The description of the property. The string is UTF-8 encoded."
+    },
+    "type": {
+      "type": "string",
+      "enum": [
+        "BOOLEAN",
+        "INT8",
+        "UINT8",
+        "INT16",
+        "UINT16",
+        "INT32",
+        "UINT32",
+        "INT64",
+        "UINT64",
+        "FLOAT16",
+        "FLOAT32",
+        "FLOAT64",
+        "BLOB",
+        "STRING",
+        "ENUM",
```

Comment:
Are we going to update this to remove the enum field like we did in the metadata spec?